tensorboard/webapp/widgets/experiment_alias/experiment_alias_component.scss

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.alias-number {
+  @include tb-theme-foreground-prop(border, border, 1px solid);
+  border-radius: 2px;
+  margin-right: 2px;
+}
+
+:host {
+  display: inline-flex;
+  align-items: baseline;
+}
+
+span {
+  background-color: inherit;
+  color: currentColor;
+  font: inherit;
+  padding: 0;
+}
```

Comment:
All of these properties/values are default values for `span`. Maybe drop it.

Reply:
Done.